src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -77,7 +77,12 @@ const dialogStyles: Partial<IModalStyles> = {
     main: {
         width: '640px !important',
         maxWidth: 'unset !important',
-        minHeight: 'fit-content'
+        minHeight: 'fit-content',
+        overflow: 'visible'
+    },
+    scrollableContent: {
+        overflow: 'visible',
+        '> div:first-child': { overflow: 'visible' }
```

Comment:
Does this code make the overflow of the dropdown options actually show up? I believe we have been using Fluent's Callout in other places, if this works in the other places it could improve performance.

Reply:
"Does this code make the overflow of the dropdown options actually show up" - yep that is why I did that. I am using Dialog fluent component in EnvironmentPicker and there is no prop to control overflow hidden that is why I did it with styling. 

You mentioned "callout" component, it seems there is a prop for Callout fluent component that you can use: `hideOverflow`. In fact, I noticed that TwinPropertySearchDropdown defines a custom Menu for react-select component where the menu is a Callout fluent component, I think in that case "hideOverflow" could be set to false to show the overflow maybe since Callout is rendered outside of the parent component in DOM? But I didn't use a custom menu in my case, so I achieve this with styling.